src/persistence/db/upgrades/to10.h

```diff
@@ -0,0 +1,55 @@
+/*
+    Copyright © 2022 by The qTox Project Contributors
+
+    This file is part of qTox, a Qt-based graphical interface for Tox.
+
+    qTox is libre software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    qTox is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with qTox.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "src/persistence/db/rawdatabase.h"
+
+#include <QVector>
+
+class ToxPk;
+
+namespace to10
```

Comment:
Pretty odd global namespace IMO. I think this should be like dbto10 or db::to10 or dbupgrade::to10.

Reply:
Agreed, changed "to10" to "DbTo10", "peersToAuthors" to "PeersToAuthors", "peersToChats" to "PeersToChats" and renamed the file to dbto10.cpp/.h.